unit_test/batched/Test_Batched.hpp

```diff
@@ -7,6 +7,9 @@
 #include "Test_Batched_SerialGemm.hpp"
 #include "Test_Batched_SerialGemm_Real.hpp"
 #include "Test_Batched_SerialGemm_Complex.hpp"
+#include "Test_Batched_BatchedGemm.hpp"
```

Comment:
@e10harvey Since we have many batched gemm tests now for functor level and host level, I would suggest renaming these files to avoid confusion. Maybe, "Test_Batched_BatchedGemm.hpp" --> "Test_Batched_HostLevelGemm.hpp"

Reply:
Thanks, @vqd8a. HostLevelGemm is mislead to me since the interface may run on device. Are you open to "Test_Batched_BatchedGemm.hpp" --> "Test_Batched_BatchedSerialGemm.hpp" instead?

Reply:
@e10harvey, yes, host level is misleading. More accurate is front-level batched gemm. But I am fine with "Test_Batched_BatchedSerialGemm.hpp" . BTW, do you have plan to have front-level interface for batched gemm using shared memory?

Reply:
Ok. No, the BatchedSerialGemm interface will be updated to use shared memory if that gives better performance for the inputs and underlying hardware. Thinking more about the name more now, BatchedSerialGemm may be misleading as well since the interface may use hierarchical parallelism for certain inputs. Are you open to leaving it named "Test_Batched_BatchedGemm.hpp"?

Reply:
One Test*.hpp file per interface: BatchedGemm, SerialBatchedGemm, etc. sounds reasonable to me.